src/spin.jl

```diff
@@ -34,6 +34,18 @@ SpinBasis(spinnumber::Int) = SpinBasis(convert(Rational{Int}, spinnumber))
 
 ==(b1::SpinBasis, b2::SpinBasis) = b1.spinnumber==b2.spinnumber
 
+"""
+    identityoperator(b::SpinBasis)
+
+Pauli ``I`` operator for the given Spin basis.
+"""
+function identityoperator(b::SpinBasis)
```

Comment:
Why is this function necessary? The generic `identityoperator` method works on `SpinBasis` without it

Reply:
i think i realized that after writing this function, as i don't seem to use it...